src/plugins/intel_cpu/src/perf_count.h

```diff
@@ -49,8 +55,135 @@ class PerfHelper {
     ~PerfHelper() { counter.finish_itr(); }
 };
 
+#else // CPU_DEBUG_CAPS
```

Comment:
It seems a bit cleaner to have two separate files, like perf_count.h and perf_count_debug_caps.h